spacy/lang/sl/tokenizer_exceptions.py

```diff
@@ -0,0 +1,260 @@
+from typing import Dict, List
+from ..tokenizer_exceptions import BASE_EXCEPTIONS
+from ...symbols import ORTH, NORM
+from ...util import update_exc
+
+_exc: Dict[str, List[Dict]] = {}
+
+_other_exc = {
+    "...": [{ORTH: "."}, {ORTH: "."}, {ORTH: "."}],
+    "t.i.": [{ORTH: "t.", NORM: "tako"}, {ORTH: "i.", NORM: "imenovano"}],
+    "t.j.": [{ORTH: "t.", NORM: "to"}, {ORTH: "j.", NORM: "je"}],
+    "T.j.": [{ORTH: "T.", NORM: "to"}, {ORTH: "j.", NORM: "je"}],
+    "d.o.o.": [{ORTH: "d.", NORM: "družba"}, {ORTH: "o.", NORM: "omejeno"}, {ORTH: "o.", NORM: "odgovornostjo"}],
+    "D.O.O.": [{ORTH: "D.", NORM: "družba"}, {ORTH: "O.", NORM: "omejeno"}, {ORTH: "O.", NORM: "odgovornostjo"}],
+    "d.n.o.": [{ORTH: "d.", NORM: "družba"}, {ORTH: "n.", NORM: "neomejeno"}, {ORTH: "o.", NORM: "odgovornostjo"}],
+    "D.N.O.": [{ORTH: "D.", NORM: "družba"}, {ORTH: "N.", NORM: "neomejeno"}, {ORTH: "O.", NORM: "odgovornostjo"}],
+    "d.d.": [{ORTH: "d.", NORM: "delniška"}, {ORTH: "d.", NORM: "družba"}],
+    "D.D.": [{ORTH: "D.", NORM: "delniška"}, {ORTH: "D.", NORM: "družba"}],
+    "s.p.": [{ORTH: "s.", NORM: "samostojni"}, {ORTH: "p.", NORM: "podjetnik"}],
+    "S.P.": [{ORTH: "S.", NORM: "samostojni"}, {ORTH: "P.", NORM: "podjetnik"}],
+    "l.r.": [{ORTH: "l.", NORM: "lastno"}, {ORTH: "r.", NORM: "ročno"}],
+    "le-te": [{ORTH: "le"}, {ORTH: "-"}, {ORTH: "te"}],
+    "Le-te": [{ORTH: "Le"}, {ORTH: "-"}, {ORTH: "te"}],
+    "le-ti": [{ORTH: "le"}, {ORTH: "-"}, {ORTH: "ti"}],
+    "Le-ti": [{ORTH: "Le"}, {ORTH: "-"}, {ORTH: "ti"}],
+    "le-to": [{ORTH: "le"}, {ORTH: "-"}, {ORTH: "to"}],
+    "Le-to": [{ORTH: "Le"}, {ORTH: "-"}, {ORTH: "to"}],
+    "le-ta": [{ORTH: "le"}, {ORTH: "-"}, {ORTH: "ta"}],
+    "Le-ta": [{ORTH: "Le"}, {ORTH: "-"}, {ORTH: "ta"}],
+    "le-tega": [{ORTH: "le"}, {ORTH: "-"}, {ORTH: "tega"}],
+    "Le-tega": [{ORTH: "Le"}, {ORTH: "-"}, {ORTH: "tega"}]
+}
+
+_exc.update(_other_exc)
+
+
+for exc_data in [
```

Comment:
In general the `NORM` value has to be a single form, It's fine if that form consists of a phrase, like `to je`, but it should be possible to replace the abbreviation with the `NORM` form in any text and for the text to still make sense. Some of these `NORM` forms are more like dictionary entries, either because they have multiple meanings (`agronomija, agrotehnika`) or grammatical forms (`angleški, angleško`) or because they contain words that are not directly part of the abbreviation (`prehodna raba (glagola)`). 

In general if it's not possible to assign a single `NORM` form to an abbreviation, it's probably best to leave it without one.

Reply:
I changed everything to a single form. In the future I will also try to add more `NORM` forms.